SystemManager/Src/SystemManager.cpp

```diff
@@ -11,11 +11,20 @@
 #include "sbus_defines.h"
 #include "sbus_receiver.hpp"
 #include "tim.h"
+#include "cmsis_os.h"
+#include "FreeRTOS.h"
+#include <iostream>
 
+extern "C" {
+  #include "app_fatfs.h"
+  #include "log_util.h"
+}
 
 #define TIMEOUT_CYCLES 250000 // 25k = 1 sec fro testing 10/14/2023 => 250k = 10 sec
 #define TIMOUT_MS      10000 // 10 sec
 
+static TaskHandle_t taskHandles[3];
```

Comment:
suggest splitting into 3 handles explicitly named after each task for clarity. if kept as an array, suggest adding a comment to layout which index holds which task.